canonical/ubuntu.com
# WD-10729 - update contact us forms

## Done

Updated multiple contact forms. Please see more details in the Jira ticket.  

## QA

- demos:
- https://ubuntu-com-13918.demos.haus/support/contact-us?product=support-overview
- https://ubuntu-com-13918.demos.haus/contact-us/form?product=pro
- https://ubuntu-com-13918.demos.haus/security/esm#get-in-touch
- https://ubuntu-com-13918.demos.haus/pricing/pro#get-in-touch
- https://ubuntu-com-13918.demos.haus/ceph/consulting#get-in-touch
- https://ubuntu-com-13918.demos.haus/core/contact-us
- https://ubuntu-com-13918.demos.haus/server/contact-us
- https://ubuntu-com-13918.demos.haus/openstack/contact-us
- https://ubuntu-com-13918.demos.haus/kernel/real-time/contact-us
- https://ubuntu-com-13918.demos.haus/aws/contact-us
- https://ubuntu-com-13918.demos.haus/aws#get-in-touch
- https://ubuntu-com-13918.demos.haus/landscape/pricing#get-in-touch
- https://ubuntu-com-13918.demos.haus/landscape/features#get-in-touch
- https://ubuntu-com-13918.demos.haus/desktop/organisations#get-in-touch
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the forms are correct according to copy docs

## Issue / Card
[WD-10729](https://warthogs.atlassian.net/browse/WD-10729)

[WD-10729]: https://warthogs.atlassian.net/browse/WD-10729?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
